packages/core/__tests__/command.test.ts

```diff
@@ -0,0 +1,79 @@
+import * as command from '../src/command'
+import * as os from 'os'
+
+/* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
this is so beforeAll can reference `originalWriteFunction`. note, `core.test.ts` does the same thing so there was a precedent for disabling this already